rascal-vscode-extension/test-workspace/test-project/src/main/rascal/SingleModuleRenameTest.rsc

```diff
@@ -0,0 +1,31 @@
+module SingleModuleRenameTest
```

Comment:
similar for this file, do we need it? or can we move it to the `rascal-lsp/test` directory?